### PR TITLE
Support nested experiment more friendly

### PR DESCRIPTION
**Pull request by Lijiaoa**
fix issue https://github.com/microsoft/nni/issues/2513

now only support ` choice ` type

demo: http://13.77.78.63:9090/detail

### CODE REVIEW

**Comment by Lijiaoa** [comment]
layer0 yAxis had have ` Empty ` choice, delete ` null `;
yAxis ` pooling_size ` had renamed ` Avg_pool_pooling_size `

File: src/webui/src/components/trial-detail/Para.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import ReactEcharts from 'echarts-for-react';
 import { filterByStatus } from '../../static/function';
 import { EXPERIMENT } from '../../static/datamodel';
-import { Stack, PrimaryButton, Dropdown, IDropdownOption, } from 'office-ui-fabric-react'; // eslint-disable-line no-unused-vars
+import { Stack, PrimaryButton, Dropdown, IDropdownOption } from 'office-ui-fabric-react'; // eslint-disable-line no-unused-vars
```
Comment (liuzhe-lz) [inline src/webui/src/components/trial-detail/Para.tsx:5]: Why is the `eslint-disable` needed?

File: src/webui/src/components/trial-detail/Para.tsx [hunk 2]
```diff
@@ -79,23 +83,30 @@ class Para extends React.Component<ParaProps, ParaState> {
...
-                for (let i = 0; i < dimName.length; i++) {
-                    if ('type' in parallelAxis[i]) {
-                        temp.push(eachTrialParams[item][dimName[i]].toString());
-                    } else {
-                        // default metric
-                        temp.push(eachTrialParams[item][dimName[i]]);
+            let paraYdata: number[][] = [];
+            const { isNested } = this.state;
+            if (isNested === false) {
+                Object.keys(eachTrialParams).map(item => {
```
Comment (liuzhe-lz) [inline src/webui/src/components/trial-detail/Para.tsx:89]: I think `for ... of` loop should be prefered if the return value is not used.

File: src/webui/src/components/trial-detail/Para.tsx (hunk 1 above)
Comment (Lijiaoa) [reply src/webui/src/components/trial-detail/Para.tsx:5]: Will delete eslint note now.

File: src/webui/src/components/trial-detail/Para.tsx (hunk 2 above)
Comment (ultmaster) [reply src/webui/src/components/trial-detail/Para.tsx]: Agreed. foreach(item => ... is also better

File: src/webui/src/components/trial-detail/Para.tsx [hunk 3]
```diff
@@ -79,23 +83,30 @@ class Para extends React.Component<ParaProps, ParaState> {
...
+                    const temp: number[] = [];
+                    for (let i = 0; i < dimName.length; i++) {
+                        // if ('type' in parallelAxis[i]) {
```
Comment (ultmaster) [inline src/webui/src/components/trial-detail/Para.tsx]: Remove unused comments here.

Comment (Lijiaoa) [reply src/webui/src/components/trial-detail/Para.tsx]: ok,

**Pull request merged**